### DISCUSSION TITLE
Miru context doesn't allow to turn replying off?

### DISCUSSION THREAD

**Discussion by tristendillon** [hypergonial/hikari-miru#34] [Q&A] [answered]
Is there a way to use the Miru context to turn replys to false in ctx.respond?

I've tried it like how you would with lightbulb's context, and it didn't work. Is this intended?

**Comment by novanai** [comment]
would you be able to provide an image to show what you're describing?

**Comment by tristendillon** [comment]
Heres a little photo of the thing im working on, is there a way to stop the bot from replying on miru's end?
![image](https://user-images.githubusercontent.com/96560330/210121253-9c416656-6ce5-4ad8-867e-c1842ee71e35.png)

**Reply by theonlydoublee** [reply]
Discord requires interactions to be responded to or it will say the application did not respond.  You can respond to the user with an ephemeral response then create a new message

**Comment by hypergonial** [comment, marked as answer]
This is a Discord limitation. You cannot hide this when creating a response unless using the workaround outlined above, due to how interactions work on Discord.
